client/packages/system/src/Patient/ListView/ListView.tsx

```diff
@@ -9,8 +9,8 @@ import {
   Fade,
   NothingHere,
   createQueryParamsStore,
-  useFormatDateTime,
-  ColumnAlign,
+  // useFormatDateTime,
```

Comment:
can you add the DOB back in? It's working and wanted, and I'm not sure why it is removed